### ISSUE TITLE
Keystroke not recognized on Windows

### ISSUE THREAD

**Issue by jdospina95** [iFgR/vue-shortkey#79]
When pressing `alt` while using Windows the keystroke is not recognized, however on Linux it is recognized. I am trying to do a copy paste in my app by pressing the `ctrl + alt + c` for copy and `ctrl + alt + v` for paste but it is not currently working since vue-shortkey is not recognizing the key press. I tried switching between keyboard layouts but it seems like the error is not related to that. I am using US-layout.

Working as expected on Linux but not on Windows, any idea?

**Comment by iFgR** [comment]
I'll try it on Windows. I'm using linux.

**Comment by iFgR** [comment]
@mdix #80 solve this?

**Comment by mdix** [comment]
@iFgR Nope, not solved by this. The linked PR just solves the issue that in IE11 the `Event` object can not be used as a constructor. A possible fix would be to go with keycodes as they're consistent through all browsers. Just let me know if you're short on time, I'd invest some time in the implementation.

**Comment by iFgR** [comment]
@mdix I'd appreciate so much if you could help me!
I'm changing the city and it's giving me so much time.
